apps/google-analytics-4/frontend/src/components/config-screen/assign-content-type/AssignContentTypeSection.tsx

```diff
@@ -59,9 +62,10 @@ const AssignContentTypeSection = (props: Props) => {
   const sdk = useSDK<AppExtensionSDK>();
 
   useEffect(() => {
+    if (parameters.useTrailingSlash) setUseTrailingSlash(parameters.useTrailingSlash);
```

Comment:
might be worth putting `setLoadingCntentTypes(true)` here as there are useEffects in place that may re-trigger this logic.